platform/vs/docker-sonic-vs/Dockerfile.j2

```diff
@@ -60,6 +60,8 @@ RUN pip install crontab
 # Install dependencies for Dynamic Port Breakout
 RUN pip install xmltodict==0.12.0
 RUN pip install jsondiff==1.2.0
+RUN pip install click==7.0.0
```

Comment:
Why the need for this? Click is already installed as a .deb on line 36. We already had a similar discussion in https://github.com/Azure/sonic-buildimage/pull/3907

Reply:
Yes, you are right. Forgot to remove while cherry-picking from the local machine. Removed it as requested.